### PR TITLE
DockerHub Image Documentation

### PR DESCRIPTION
**Pull request by dmccoystephenson**
## Changes
Documentation for using the image has been added to `docs/dockerhub.md`.

### CODE REVIEW

**Review by drewjj — APPROVED** [review]
Looks good

File: docs/dockerhub.md [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+> docker pull usdotjpoode/jpo-ode:latest
+
+## Required environment variables
+The image expects the following environment variables to be set:
+- DOCKER_HOST_IP
+- DOCKER_SHARED_VOLUME
+
+## Direct Dependencies
+The ODE will fail to start up if the following containers are not already present:
+- Kafka
```
Comment (payneBrandon) [inline docs/dockerhub.md:20]: We can configure this to use confluent rather than a kafka container right? Maybe reword this to say:
```
dependencies
- Kafka or Confluent and related requirements
```

File: docs/dockerhub.md [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+
+## Indirect Dependencies
+Some functionality will be unreachable without the participation of the following programs (except by directly pushing to kafka topics):
+- asn1_codec
+- jpo-cvdp
+- jpo-security-svcs
+- jpo-sdw-depositor
+- jpo-s3-deposit
+
+## Example docker-compose.yml with direct dependencies:
```
Comment (payneBrandon) [inline docs/dockerhub.md:31]: Let's swap this header to a "Configuration" and add in a note in here about "for further configuration options see GitHub repo with the link to it on "GitHub". Then we can subheader the example doc below

**Review by payneBrandon — CHANGES_REQUESTED** [review]
Looking great, just a couple tweaks
